app/helpers/tracks_helper.rb

```diff
@@ -29,8 +29,9 @@ def bordered_track_icon(color, track, options={})
   end
 
   def track_icon(type, color, track, options={})
+    options[:alt] = "Track #{track.title}"
     options[:onerror] = "this.onerror=null;this.src='https://assets.exercism.io/tracks/default-#{type}-#{color}.png'"
-    image_tag(track.send("#{type}_#{color}_icon_url"), options)
+    image_tag track.send("#{type}_#{color}_icon_url"), options
```

Comment:
Could we stick to using brackets in methods in `helpers` pls? That will keep it consistent with the rest of the codebase. I'm aware it's a personal preference, but I prefer the explicity of it. In the HAML, on the other hand, I prefer not. 

At some point soon when I get chance, I will write these "style" things up as part of the contributing guide to this repo, and would also really appreciate your input on adding rules for how to keep things accessible beyond this initial PR.

Reply:
Sure, I couldn't figure out the style preferences for the project that's why I probably mixed it up a bit. Brackets for methods, space after equal sign in HAML templates `=image` or `= image`, single or double quotes in templates and code. Can you please give me very brief instructions?

Reply:
I will write a quick document with some notes in for this and then you can PR instructions for the a11y to it too :)